File: angular-web2-app/src/app/entities/resenje/resenje.ts
export class Resenje {
    NazivUzroka:string;
    NazivPoduzroka:string;
    NazivTipKonstrukcije:string;
    NazivTipMaterijala:string;

    constructor(nazivUzroka:string, nazivPoduzroka:string, nazivTipKonstrukcije:string, nazivTipMaterijala:string) {
        this.NazivUzroka = nazivUzroka;
        this.NazivPoduzroka = nazivPoduzroka;
        this.NazivTipKonstrukcije = nazivTipKonstrukcije;
        this.NazivTipMaterijala = nazivTipMaterijala;
    }
}

File: api/webAPI/WebApplication1/Models/Resenje.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.Models
{
    public class Resenje
    {
        public int idResenje { get; set; }
        public string NazivUzroka { get; set; }
        public string NazivPoduzroka { get; set; }
        public string NazivTipKonstrukcije { get; set; }
        public string NazivTipMaterijala { get; set; }
    }
}